include/aos/sm/servicemanager.hpp

```diff
@@ -82,7 +82,8 @@ struct ServiceData {
     bool operator==(const ServiceData& data) const
     {
         return mServiceID == data.mServiceID && mProviderID == data.mProviderID && mVersion == data.mVersion
-            && mImagePath == data.mImagePath && mCached == data.mCached && mSize == data.mSize && mGID == data.mGID;
+            && mImagePath == data.mImagePath && data.mManifestDigest == mManifestDigest && mTimestamp == data.mTimestamp
```

Comment:
Not sure mTimestamp comparation is a good idea
Timestamp is usually the time object was created.
Only DB has the proper date.

Reply:
we use this comparation in DB, and also we should be sure that object is equal. @al1img  need your opinion

Reply:
Done